### DISCUSSION TITLE
Placement of activation buttons

### DISCUSSION THREAD

**Discussion by trueberryless** [trueberryless-org/starlight-view-modes#2] [Polls]
User eXperience can be hard to achieve in an intuitive way. Therefore I'd like to get as many votes as possible so I can estimate where the initial activation of the different modes should be placed. Please vote down below and if you have a **good idea** let me know in the comments...

Currently the buttons are placed here:

![image](https://github.com/trueberryless/starlight-view-modes/assets/99918022/95069c38-8534-4e78-9070-730925f84933)

**Comment by abdulhakim-alshanqiti** [comment]
shouldn't it be next to the search icon? there is another plugin that allows for links in that place it may help 
![Screenshot_20240629_231231](https://github.com/trueberryless/starlight-view-modes/assets/136203274/431e33ee-12ec-41c5-bf69-d89bf1a21e26)

![Screenshot_20240629_231513](https://github.com/trueberryless/starlight-view-modes/assets/136203274/edb8f793-8f3a-4414-9702-53811b9ee89a)

also i think it should be next to these places its


wait i think you already did it!

**Reply by trueberryless** [reply]
I'm not planning to place a switch in the top bar on mobile devices because the switch is already hidden behind the context window (menu) where social links are also displayed.

If you can maybe link to the other plugin which enables links in that place, I'll maybe consider implementing an option for this after concentrating on the new presentation mode.

**Comment by robmunger** [comment]
The original location looks great. It is unobtrusive and will be found by the power users who will really appreciate it.

Other locations could be added as options set in astro.config.mjs, but I probably wouldn't personally use them.

**Reply by abdulhakim-alshanqiti** [reply]
honstly after thinking about it i like the way the plugin looks now...it would be better if the view modes on table of contents would be before everything else so in big pages the view modes could be reached easily

also an option to change the svg would be nice i currently clone the plugin and changed the svg

**Reply by trueberryless** [reply]
yeah, before the table of contents is a good idea. I put it down below because I was inspired by the Astro docs. Unfortunately I'm currently not working much anymore on this plugin, but I'm looking forward to having time for it again...

**Reply by robmunger** [reply]
Gotcha. Personally, I do like it after the ToC, as it's a bit more innocuous, but I could live with it up top.
